package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "systemjs-builder",
-  "version": "0.15.35",
+  "version": "0.15.36",
```

Comment:
0.16 will be fine. I'm happy to help them get aurelia-bundler working with 0.16 and then 0.15 support wont matter to me!